app/src/app/components/note-viewer/elements/drawing/DrawingElementComponent.tsx

```diff
@@ -7,12 +7,34 @@ import { Input, Row } from 'react-materialize';
 import stringify from 'json-stringify-safe';
 import * as FullScreenService from '../../../../services/FullscreenService';
 
-type Touch = {
+class Touch {
```

Comment:
Can you please move the auxiliary class definitions to the bottom of the file?

Reply:
Done